docs/api-reference/json.rst

```diff
@@ -17,7 +17,7 @@ Project
 
     .. code:: http
 
-        GET /pypi/pip/json HTTP/1.1
+        GET /pypi/visidata/json HTTP/1.1
```

Comment:
Any particular reason why we're changing the example project here?

Reply:
I suggested this. I think that using `pip` as the example project is apt to cause at least a little confusion when less experienced developers run into problems and try to reason through what has gone wrong.

Reply:
Can we use [`sampleproject`](https://pypi.org/project/sampleproject/) instead then?

Reply:
Great idea!